cookbooks/aws-parallelcluster-config/resources/manage_ebs.rb

```diff
@@ -131,22 +144,23 @@
 
   # Mount each volume
   dev_path = [] # device labels
-  dev_uuids = [] # device uuids
 
   vol_array.each_with_index do |volumeid, index|
     dev_path[index] = "/dev/disk/by-ebs-volumeid/#{volumeid}"
-    dev_uuids[index] = get_uuid_for_unmount(shared_dir_array[index])
 
     # Unmount and remove volume from /etc/fstab
-    mount shared_dir_array[index] do
-      device(DelayedEvaluator.new { dev_uuids[index] })
-      fstype(DelayedEvaluator.new { node['cluster']['volume_fs_type'] })
-      device_type :uuid
-      options "_netdev"
-      pass 0
-      action %i(unmount disable)
+    execute 'unmount ebs' do
```

Comment:
Here for EBs and everywhere else for EFS/FSx: what about putting the unmounting steps before the mounting steps?
I say so because:
1. lazy unmounting actions operations may have better chances to succeed w.r.t. mounting operations
2. since mounting operations may require external resources to be ready to be mounted (FS, networking,...) better to give some extra time to complete the setup if we can